js/sitnlayers.js

```diff
@@ -121,11 +125,25 @@
       _drawFillColor;
       let _mainbar;
       sitnLayers.setDrawStyle({});
+
+      if (_wmslayers) {
+        sitnLayers.sitnWMSLayers.push(
+          new ol.layer.Image({
+            extent: _extent,
+            source: new ol.source.ImageWMS({
+              url: _WMSurl,
+              params: { 'LAYERS': _wmslayers.join(',') },
+              serverType: 'mapserver'
+            })
+          })
+        )
+      }
       _map = new ol.Map({
         target: _target,
         layers: [
           sitnLayers.sitnCurrentBaseLayer,
           sitnLayers.sitnDrawLayer,
+          ...sitnLayers.sitnWMSLayers,
```

Comment:
Why are there three dots here?
```suggestion
          sitnLayers.sitnWMSLayers,
```

Reply:
This is the spread operator https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Op%C3%A9rateurs/Syntaxe_d%C3%A9composition

Reply:
OK, got it, thanks !